common/services/prismic/exhibitions.js

```diff
@@ -156,6 +170,45 @@ export async function getExhibitions(req: Request, id: string): Promise<Paginate
   };
 }
 
+export async function getExhibitionsComingUp(req: Request) {
+  const order = 'asc';
+  const predicates = [
+    Prismic.Predicates.dateAfter(startField, london().toDate())
+  ];
+
+  const paginatedResults = await getExhibitions(req, {
+    predicates, order
+  });
+
+  return paginatedResults;
+}
+
+export async function getExhibitionsCurrent(req: Request) {
+  const order = 'desc';
+  const predicates = [
+    Prismic.Predicates.dateBefore(startField, london().toDate()),
```

Comment:
Should these be the other way round? `dateBefore(endField)` and `dateAfter(startField)`?